lke_clusters.go

```diff
@@ -284,3 +291,31 @@ func (c *Client) DeleteLKEClusterServiceToken(ctx context.Context, clusterID int
 	e := formatAPIPath("lke/clusters/%d/servicetoken", clusterID)
 	return doDELETERequest(ctx, c, e)
 }
+
+// GetLKEClusterAPLConsoleURL gets the URL of this cluster's APL installation if this cluster is APL-enabled.
+func (c *Client) GetLKEClusterAPLConsoleURL(ctx context.Context, clusterID int) (string, error) {
+	cluster, err := c.GetLKECluster(ctx, clusterID)
+	if err != nil {
+		return "", err
+	}
+
+	if cluster.APLEnabled {
+		return fmt.Sprintf("https://console.lke%v.akamai-apl.net", cluster.ID), nil
+	}
+
+	return "", nil
+}
+
+// GetLKEClusterAPLHealthCheckURL gets the URL of this cluster's APL health check endpoint if this cluster is APL-enabled.
+func (c *Client) GetLKEClusterAPLHealthCheckURL(ctx context.Context, clusterID int) (string, error) {
+	cluster, err := c.GetLKECluster(ctx, clusterID)
+	if err != nil {
+		return "", err
+	}
+
+	if cluster.APLEnabled {
+		return fmt.Sprintf("https://auth.lke%v.akamai-apl.net/ready", cluster.ID), nil
```

Comment:
```suggestion
		return fmt.Sprintf("https://auth.lke%d.akamai-apl.net/ready", cluster.ID), nil
```
Let's use `%d` just to be safe.